loopialib/client.py

```diff
@@ -22,10 +22,10 @@ def _parse_status_code(response):
 
 
 class Loopia(object):
-    base_url = "https://api.loopia.se/RPCSERV"
     encoding = "utf-8"
 
-    def __init__(self, user, password):
+    def __init__(self, user, password, domain = 'se'):
+        self.base_url = f"https://api.loopia.{domain}/RPCSERV"
```

Comment:
This breaks the test-api at the bottom of this file. However it does seem like that service is no longer available. Please remove that as part of this merge request.